.travis.yml

```diff
@@ -101,6 +101,7 @@ env:
     - TRAVIS_FLAVOR=win32_event_log
     - TRAVIS_FLAVOR=kafka_consumer FLAVOR_VERSION=0.10.1.0-1
     - TRAVIS_FLAVOR=kafka FLAVOR_VERSION=0.10.1.0-1
+    - TRAVIS_FLAVOR=docker_daemon FLAVOR_VERSION=latest
```

Comment:
So I wanted to talk to you about this.... I was wondering if docker, kubernetes, etc should get their own "suite" so to speak, we could bundle them all into the same test "flavor"